src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,38 @@
+package duke.task;
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        isDone = false;
+    }
+
+    public void markDone() {
+        System.out.println("Nice! I've marked this task as done:");
+        isDone = true;
```

Comment:
maybe you can make isDone = "X" or a " " since it would slightly shorten your code. Whenever you want to check isDone is true, you can just check if isDone == "X" instead. not necessary for functionality!